doc/release/release_2.6.rst

```diff
@@ -491,7 +491,7 @@ Merged PRs
 - Improve intersection function (#4588)
 - Pygraphviz choco (#4583)
 - Add prominent group algorithm (#4560)
-- Add partition_quality to compute coverage and performance  (coverage and perfor… (#4536)
+- Add partition_quality to compute coverage and performance  (coverage and perform… (#4536)
```

Comment:
Wellllll, technically not a typo as it just the text from the PR title https://github.com/networkx/networkx/pull/4536

Reply:
;-) indeed -- might catch some other similar ones too later on.